container-images/mds-web-sockets/webpack.config.ts

```diff
@@ -1,3 +1,3 @@
 import webpack from '@mds-core/mds-webpack-config'
 
-export default webpack.Bundle('./index.ts').UsingDefaultConfig()
+export default webpack.Bundle('../../packages/mds-web-sockets/launch_server.ts').UsingDefaultConfig()
```

Comment:
1. Rename `launch_server.ts` to just `server.ts`
2. Change DockerFile to `ENTRYPOINT ["/sbin/tini", "node", "--no-deprecation", "server.js"]`